### ISSUE TITLE
Edit Mode Move along Normals

### ISSUE THREAD

**Issue by seularts** [cgfasttrack/lazyviewport#4]
Normally in edit mode I double tap G to move along the Normals, but with the plugins it only lets me to move along the axis. Is there a way to make it work?

**Comment by seularts** [comment]
Sorry, I wanted to mean along edges, not normals!

**Comment by cgfasttrack** [comment]
Yes this is not currently implemented, defiantly will do this in the future. Its not looking like I will have time for this for quite some though. 

In the meantime, **move along vert hotkey is shift + v** and then I usually have **"edge slide" mapped to a quick hotkey (the "Q" hotkey).** 

We have bigger plans for the lazy viewport as well. We want to get the rotation hotkeys in there as well along with a few other additional hotkeys the standard g,r,s hotkeys have.

**Comment by seularts** [comment]
Thank you for the reply, yes Shift+V is a good alternative at the moment.
